lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -33,9 +33,9 @@ BEGIN_NAMESPACE_OPENTURNS
 static const char * ConfigurationFileName = "openturns.conf";
 #if defined OPENTURNS_HAVE_LIBXML2
 static const char * RootElementName = "openturns-configuration";
-static const char * XMLTag_value                  = "value_string";
-static const char * XMLTag_value_scalar           = "value_scalar";
-static const char * XMLTag_value_unsigned_integer = "value_unsigned_integer";
+static const char * XMLTag_value                  = "value_str";
```

Comment:
It would be better to name the variable XMLTag_value_str here, and so on.

Reply:
+1

Reply:
ok!